### PR TITLE
Add NSFileManager.UnmountVolume

### PR DESCRIPTION
**Pull request by chamons**
Requested in https://forums.xamarin.com/discussion/comment/263094 and trivial to knock out.

### CODE REVIEW

File: src/foundation.cs [hunk 1]
```diff
@@ -10980,6 +10980,10 @@ partial interface NSFileManager {
 #else
 		bool GetRelationship (out NSURLRelationship outRelationship, NSUrl directoryURL, NSUrl otherURL, out NSError error);
 #endif
+
+		[NoWatch][NoTV][NoiOS][Mac (10, 11)]
+		[Export ("unmountVolumeAtURL:options:completionHandler:")]
+		void UnmountVolume (NSUrl url, NSFileManagerUnmountOptions mask, Action<NSError> completionHandler);
```
Comment (rolfbjarne) [inline src/foundation.cs]: Would it make sense to `[Async]` this?

Comment (chamons) [inline src/foundation.cs]: Yes. Let me add that, I had not considered it.

Comment (spouliot) [inline src/foundation.cs]: intro should have warned you :)

Comment (chamons) [inline src/foundation.cs]: I did this in a "let's just build src and PR it" rush. I figured I'd at least get test failures before you guys jumped all over it.

Yes, I should feel bad apparently.

Comment (spouliot) [inline src/foundation.cs]: _because I like my code public and my mistakes private_ -> https://github.com/spouliot/Touch.Unit ;-)

Comment (chamons) [inline src/foundation.cs]: https://en.wikipedia.org/wiki/List_of_burn_centers_in_the_United_States#Texas

**Comment by chamons** [comment]
Test failure is known: https://bugzilla.xamarin.com/show_bug.cgi?id=52968

**Pull request merged**